src/utils.ts

```diff
@@ -49,9 +49,9 @@ export async function setUpNpmConfig (nodeCtx: NodeContext, userConfig: NpmConfi
     audit: false,
     fund: false,
     noproxy: 'registry.npmjs.org',
-    cafile: process.env.CA_FILE || null,
+    cafile: process.env.NPM_CONFIG_CAFILE || null,
     'package-lock': false,
-    'strict-ssl': true,
+    'strict-ssl': false,
```

Comment:
I understand the change for `cafile: process.env.NPM_CONFIG_CAFILE || null,`, since chef actually sets `NPM_CONFIG_CAFILE` and not `CA_FILE` 👍 
However, why then is `'strict-ssl': false,` needed?

Reply:
Oh my typo..